### PR TITLE
Give me the peac gun

### PR DESCRIPTION
**Pull request by Alberyk**
This pr adds a new var to projectiles: anti_materiel_potential.
anti_materiel_potential is a multiplier of how much damage a projectile in question will do to mechas, structures, and ipcs. This way we can have anti-materiel weapons that are not insanely powerful against regular mobs. Besides adding this to the hand cannon, the unathi slugger, and the anti-materiel rifle, this pr also adds two new weapons:

Point entry anti-materiel cannon (peac): a futuristic weapon that fires anti-materiel missiles. While it holds a single round, it can be reloaded.

Adhomian recoilless rifle: a one-use adhomian surplus anti-tank gun.

### CODE REVIEW

File: code/modules/mob/living/living_defense.dm [hunk 1]
```diff
@@ -40,6 +40,8 @@
 	//Armor
 	var/damage = P.damage
 	var/flags = P.damage_flags()
+	if(ismech(src))
+		damage = P.damage * P.anti_materiel_potential
```
Comment (NonQueueingMatt) [inline code/modules/mob/living/living_defense.dm:44]: Not a fan of this. Override bullet_act on the mech and put it at the top before calling ..().

**Review by NonQueueingMatt — CHANGES_REQUESTED** [review]

File: code/modules/projectiles/guns/projectile/cannon.dm [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
 /obj/item/gun/projectile/cannon/special_check(mob/user)
 	if(!wielded)
-		to_chat(user, "<span class='warning'>You can't fire without stabilizing \the [src]!</span>")
+		to_chat(user, SPAN_WARNING ("You can't fire without stabilizing \the [src]!"))
```
Comment (Geevies) [inline code/modules/projectiles/guns/projectile/cannon.dm:27]:
```suggestion
		to_chat(user, SPAN_WARNING("You can't fire without stabilizing \the [src]!"))
```

File: code/modules/projectiles/guns/projectile/cannon.dm [hunk 3]
```diff
@@ -43,14 +43,14 @@
...
-	fire_sound = 'sound/weapons/click.ogg'
+	fire_sound = 'sound/weapons/mortar.ogg'
 	recoil = 4
 
 	auto_eject = TRUE
 
 	is_wieldable = TRUE
 
-	desc_fluff = "The People's Republic of Adhomai is the only Adhomian faction able to master the nuclear fission. Atomic weapons were used before in the tajaran civil war, \
+	desc_fluff = "The People's Republic of Adhomai is the first  Adhomian faction able to master the nuclear fission. Atomic weapons were used before in the tajaran civil war, \
```
Comment (Geevies) [inline code/modules/projectiles/guns/projectile/cannon.dm:53]:
```suggestion
	desc_fluff = "The People's Republic of Adhomai is the first Adhomian faction able to master the nuclear fission. Atomic weapons were used before in the tajaran civil war, \
```

File: code/modules/projectiles/guns/projectile/cannon.dm [hunk 4]
```diff
@@ -62,6 +62,88 @@
 
 /obj/item/gun/projectile/nuke/special_check(mob/user)
 	if(!wielded)
-		to_chat(user, "<span class='warning'>You can't fire without stabilizing \the [src]!</span>")
+		to_chat(user, SPAN_WARNING ("You can't fire without stabilizing \the [src]!"))
```
Comment (Geevies) [inline code/modules/projectiles/guns/projectile/cannon.dm:65]:
```suggestion
		to_chat(user, SPAN_WARNING("You can't fire without stabilizing \the [src]!"))
```

File: code/modules/projectiles/guns/projectile/cannon.dm [hunk 5]
```diff
@@ -62,6 +62,88 @@
...
+		icon_state = "recoilless_rifle"
+		item_state = "recoilless_rifle"
+	else
+		icon_state = "recoilless_rifle-empty"
+		item_state = "recoilless_rifle-empty"
+
+
+/obj/item/gun/projectile/recoilless_rifle/special_check(mob/user)
+	if(!wielded)
+		to_chat(user, SPAN_WARNING ("You can't fire without stabilizing \the [src]!"))
```
Comment (Geevies) [inline code/modules/projectiles/guns/projectile/cannon.dm:102]:
```suggestion
		to_chat(user, SPAN_WARNING("You can't fire without stabilizing \the [src]!"))
```

File: code/modules/projectiles/guns/projectile/cannon.dm [hunk 6]
```diff
@@ -62,6 +62,88 @@
...
+	if(loaded.len)
+		icon_state = "peac"
+		item_state = "peac"
+	else
+		icon_state = "peac-empty"
+		item_state = "peac-empty"
+
+/obj/item/gun/projectile/peac/special_check(mob/user)
+	if(!wielded)
+		to_chat(user, SPAN_WARNING ("You can't fire without stabilizing \the [src]!"))
```
Comment (Geevies) [inline code/modules/projectiles/guns/projectile/cannon.dm:147]:
```suggestion
		to_chat(user, SPAN_WARNING("You can't fire without stabilizing \the [src]!"))
```

**Review by Geevies — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**